TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -797,6 +799,86 @@ public class TrafficManagerTool
             }
         }
 
+        public GuideHandler Guide;
+
+        public class GuideHandler {
```

Comment:
Why not in separate file?

Reply:
Usually this is not necessary at all. In the base game its SimulationStep() that is handling user events I don't know why we are using OnToolUpdate() .

This code is specific to TrafficManagerTool